### DISCUSSION TITLE
Can't deploy my project (PRR enabled)

### DISCUSSION THREAD

**Discussion by Vette1123** [vercel/community#4837] [Bug Report]
### Page to Investigate

https://vercel.com/

### Steps to Reproduce

I'm trying to delpoy my project, but i'm getting this weird error.

Error: Invariant: PPR lambda isn't streaming
![image](https://github.com/vercel/community/assets/26345237/9537cc2b-dabc-4052-b11b-df1e5c3b7bcb)

**Comment by leerob** [comment]
Are you using an experimental feature like Partial Prerendering? Can you share your code?

**Reply by steven-tey** [reply]
@Vette1123 can you try upgrading `next` to the latest `canary` to see if you're still facing this issue?

**Reply by Vette1123** [reply]
Hello @leerob , thanks for answering.

The issue happened when i enabled PPR in my next.config.js

![image](https://github.com/vercel/community/assets/26345237/12306aa1-e5f9-42bc-a3ab-9af15a055231)

Deployment is successful when i removed that, i was practising the next.js 14 example here: https://github.com/Vette1123/next-14-dashboard

How can i enable PPR in deployed version?

**Comment by jayair** [comment]
Same, here's my repo: https://github.com/jayair/nextjs-learn/tree/735db5968ec66bdc3d39407bef585a65c920639c

I'm trying out PPR with next@canary.

**Reply by jayair** [reply]
Looks like a false alarm. The next couple of deployments seemed fine!

**Comment by Gawdfrey** [comment]
I am getting the same error trying to deploy my next app on vercel. On version 14.0.3-canary.8, PPR is enabled.
Not able to share repo, so unsure how helpful this is, sorry!

**Comment by abiddraws** [comment]
I had this error , that got resolved when I removed the line `export const dynamic = "force-dynamic";` from my code.

**Comment by jahirfiquitiva** [comment]
hey @leerob , I'm facing the very same issue: `Error: Invariant: PPR lambda isn't streaming`

Here's my repo: https://github.com/jahirfiquitiva/jahir.dev/pull/68

I'm not sure what is causing it and the Vercel deployment logs don't mention it either. 😕

I'd be grateful if you could please help me out 🙌

**Reply by abiddraws** [reply]
Try after removing this dynamic property [revalidate](https://github.com/jahirfiquitiva/jahir.dev/blob/main/src/app/donate/sponsorships.tsx#L10) from all your code

**Reply by jahirfiquitiva** [reply]
@abiddraws I tried and it still failed
https://github.com/jahirfiquitiva/jahir.dev/pull/68/commits/f6a11e7e502136daf0d024044af396124de15dc4

**Comment by zaynelovecraft** [comment]
This is happening to me also, any one figure out how to fix this?

**Comment by lourd** [comment]
I just ran into this issue as well. Latest canary, `14.2.0-canary.6`. Code is here https://github.com/lourd/descioli.com/pull/45. Tried redeploying; got the same error.

**Reply by jahirfiquitiva** [reply]
What worked for me was removing `export const dynamicParams = false`
Give it a try!

**Reply by lourd** [reply]
Thanks for the suggestion! I tried it, removing `dynamicParams` altogether, and it did successfully deploy. But now when loading the pages that have PPR the server-side function crashes with an error like this:

```
Error: Expected the resume to render <u> in this slot but instead it rendered <d>. The tree doesn't match so React will fallback to client rendering.
```

This is the same error I have when doing `next start` locally, with or without `dynamicParams`. The only issue I can find that references this type of error is https://github.com/vercel/next.js/issues/60620

**Reply by lourd** [reply]
Strangely re-adding `dynamicParams=false` the application now successfully deployed 🤔 but still crashes on the routes with PPR

**Comment by me-imfhd** [comment]
I am also facing the same issue, i am using nextjs 14 experimental ppr to true, is this getting fixed anytime soon?
